zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt

```diff
@@ -62,4 +65,21 @@ class ZaakObjectResource(
         val form = zaakObjectService.getZaakObjectForm(objectUrl)
         return form?.let { ResponseEntity.ok(it) } ?: ResponseEntity.notFound().build()
     }
+
+    @GetMapping(value = ["/v1/zaakinstancelink/zaak"])
+    fun getZaakInstanceLink(
+        @RequestParam(name = "zaakInstanceUrl") zaakInstanceUrl: URI
+    ): ResponseEntity<ZaakInstanceLink>{
```

Comment:
The responses of both endpoints, contain more data than they should. I suggest to respond with a DTO

Reply:
Should i just return the "zaakInstanceLinkId", "zaakInstanceUrl", "zaakInstanceId", "documentId" and "zaakTypeUrl"? or is this still too much information?

Reply:
A `zaakInstanceUrl` and `documentId` is enough

Reply:
Alright, thank you